apps/etterlatte-beregning-asynk/src/main/kotlin/no/nav/etterlatte/beregningasynk/Application.kt

```diff
@@ -0,0 +1,16 @@
+package no.nav.etterlatte.beregningasynk
+
+import no.nav.helse.rapids_rivers.RapidApplication
+
+fun main() {
+    System.getenv().toMutableMap().apply {
+        put("KAFKA_CONSUMER_GROUP_ID", get("NAIS_APP_NAME")!!.replace("-", ""))
```

Comment:
Dette gjøres nå i alle rapid-appene våre - skulle vi hatt en felles kodesnutt for det tro?

Reply:
Liker tanken. Gir det eit forsøk.

Reply:
Sånn!